.travis.yml

```diff
@@ -43,4 +43,5 @@ deploy:
   skip_cleanup: true
   on:
     tags: true
-    branch: master
\ No newline at end of file
+    #branch: master
+    all_branches: true
```

Comment:
Why do we changed our deployment strategy to deploy on all branches?

Reply:
I can not wait until a simple commit is merged to master to deploy it to docker hub, I need it immediately, to test my work.

Reply:
I have discussed this with Frank Kammer and we think that we should try to keep the master stable. Please, do not use it for testing. With the same thought the docker hub images should contain a stable version and should not be used for testing. I suggest that we introduce docker hub images that are meant for testing and rewrite our travis file such that all builds on all branches, except master, are pushed as "dev" images. Is such a process okay for you?

Reply:
yes, looks good.